src/main/java/edu/tamu/app/service/SfxService.java

```diff
@@ -18,17 +17,21 @@
 import org.springframework.util.MultiValueMap;
 import org.springframework.web.client.RestTemplate;
 
+import edu.tamu.app.config.AppConfig;
+
 /**
  * The SfxService checks with the linkresolver to find out if the full text is available for the given url
  *
  * @author Jason Savell <jsavell@library.tamu.edu>
  *
  */
-
-
 @Service
+@EnableConfigurationProperties(AppConfig.class)
```

Comment:
Please move `@EnableConfigurationProperties` to WebServerInit or an applicable config file. It is only required once, and it appears on multiple services.